server/mscalendar/settings.go

```diff
@@ -35,6 +35,13 @@ func NewSettingsPanel(bot bot.Bot, panelStore settingspanel.PanelStore, settingS
 		store.UpdateStatusSettingID,
 		settingStore,
 	))
+	settings = append(settings, settingspanel.NewBoolSetting(
+		store.GetRemindersSettingID,
+		"Get Reminders",
+		"Do you want to get reminders for upcoming events?",
```

Comment:
0/5. Then we would also change the setting to `ReceiveReminders`
```suggestion
		"Do you want to receive reminders for upcoming events?",
```